website/docs/tutorials/08-cairo-1-support/01-project-initialization.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: Project initialization
+---
+
+# Project initialization
+
+## Creating a project
+To create a new Protostar project with cairo1 support, you will need to run the `protostar init-cairo1` command followed by the name of your project.
+
+After running the command, the following structure will be generated:
+
+```
+./your-project-name
+|-- protostar.toml
+|-- src
+|   |-- cairo_project.toml
+|   |-- lib.cairo
+|   `-- main.cairo
+`-- tests
+    `-- test_main.cairo
+```
+
+This template contains:
+
+- `src` directory
+    - `cairo_project.toml` which is needed for compilation 
+    - `main.cairo` file with one function definition
+    - `lib.cairo` file which defines the module
+- `test` directory
+    - Single test file with one test for function defined in `main.cairo`
+- `protostar.toml` containing information about the project
+
+## The protostar.toml
+Apart from the usual things you find in `protostar.toml`, there is `linked-libarires` entry which is used to find cairo 1 modules in tests and building.
+This makes it possible for you to include other modules from your dependencies, if they are correct cairo 1 modules (with their own module definition and `cairo_project.toml`).
```

Comment:
Not sure about this comma, worth asking someone knowledgeable about this.
```suggestion
This makes it possible for you to include other modules from your dependencies if they are correct cairo 1 modules (with their own module definition and `cairo_project.toml`).
```

Reply:
should be removed